langchain4j-vertex-ai-gemini/src/test/java/dev/langchain4j/model/vertexai/VertexAiGeminiChatModelIT.java

```diff
@@ -0,0 +1,74 @@
+package dev.langchain4j.model.vertexai;
+
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.UserMessage;
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.output.FinishReason;
+import dev.langchain4j.model.output.Response;
+import dev.langchain4j.model.output.TokenUsage;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static dev.langchain4j.model.output.FinishReason.LENGTH;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Disabled("To run this test, you must provide your own project and location")
+class VertexAiGeminiChatModelIT {
+
+    @Test
+    void should_generate_response() {
+
+        // given
+        ChatLanguageModel model = VertexAiGeminiChatModel.builder()
+                .project("langchain4j")
+                .location("us-central1")
+                .modelName("gemini-pro")
+                .build();
+
+        UserMessage userMessage = UserMessage.from("What is the capital of Germany?");
```

Comment:
Can you add tests for SystemMessage request?

Reply:
we will deny system message for now, will add a test for this